tune_sklearn/tune_basesearch.py

```diff
@@ -241,25 +301,39 @@ def _check_if_refit(self, attr):
             raise AttributeError("'{}' is not a valid attribute with "
                                  "'refit=False'.".format(attr))
 
-    def _check_is_fitted(self, method_name):
+    def _check_is_fitted(self, method_name, check_refit=True):
         """Helper method to see if the estimator has been fitted.
 
         Args:
             method_name (str): String of the method name called from the user.
 
+            check_refit(bool/str)=True: Whether to also check for `self.refit`
+            param. If "multimetric", will only check if `self.multimetric`
+            param is also True.
+
         Raises:
             NotFittedError: If the estimator has not been fitted.
             TypeError: If the estimator is invalid (i.e. doesn't implement
                 the sklearn estimator interface).
 
         """
         if not self.refit:
-            msg = ("This {0} instance was initialized with refit=False. {1} "
-                   "is available only after refitting on the best "
-                   "parameters.").format(type(self).__name__, method_name)
-            raise NotFittedError(msg)
-        else:
-            check_is_fitted(self)
+            if check_refit == "multimetric":
```

Comment:
Unsure if this is the best way to do it. Other ones I see are splitting this into two functions, or using an enum to define possible values for the parameter. Thoughts?